src/register/mod.rs

```diff
@@ -13,88 +13,89 @@
 #[macro_use]
 mod macros;
 
+mod addresses;
+
 // User Trap Setup
-pub mod uie;
-pub mod ustatus;
-pub mod utvec;
+//pub mod uie;
```

Comment:
Why is this all commented out?

Reply:
I get flooded with errors otherwise as I got rid of the macros. The alternative is to just outright delete everything and bring stuff back as I need it.